### PR TITLE
Add Seldon component to kf-v0.4 cluster

### PR DESCRIPTION
**Pull request by zhenghuiwang**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/296)

### CODE REVIEW

**Comment by jlewi** [comment]
@zhenghuiwang I think you need to modify 
https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/create_kf_instance.py

@gabrielwen is working on constantly redeploying the kubeflow clusters form the latest code on the branch for testing. That will use the script mentioned above.

**Comment by zhenghuiwang** [comment]
> @zhenghuiwang I think you need to modify
> https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/create_kf_instance.py
> 
> @gabrielwen is working on constantly redeploying the kubeflow clusters form the latest code on the branch for testing. That will use the script mentioned above.

Now I understand why the k8s server address keeps changing.

How do we keep track the latest server address, if I want to issue `kubectl` command to the cluster? do I have to get cluster-info by name first?

**Comment by gabrielwen** [comment]
Sorry for causing problem..  I'm using another cluster (n01) for testing now so outage shouldn't happen again!

File: py/kubeflow/testing/create_kf_instance.py [hunk 1]
```diff
@@ -113,6 +113,7 @@ def main(): # pylint: disable=too-many-locals,too-many-statements
     yaml.dump(app, hf)
 
   util.run([kfctl, "generate", "all"], cwd=app_dir)
+  util.run(["ks", "generate", "seldon", "seldon"], cwd=app_dir)
```
[the remark this answers is not in the document]
Comment (zhenghuiwang) [reply py/kubeflow/testing/create_kf_instance.py]: thanks for pointing it out.

**Comment by jlewi** [comment]
/lgtm
/approve
/hold 
/assign @gabrielwen 
@gabrielwen can you take a quick look.

**Pull request merged**